content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -158,13 +165,11 @@ $ curl \
 
 ## Show Team Access to a Workspace
 
-| Method | Path           |
-| :----- | :------------- |
-| GET | /team-workspaces/:id |
+`GET /team-workspaces/:id``
 
-### Parameters
-
-- `id` (`string: <required>`) - The ID of the team/workspace relationship.
+Placeholder | Description
```

Comment:
There's a mixture of `Placeholder` and `Parameter`. I think the latter works better since it is also the terminology internally, not sure placeholder would map as well.

Reply:
I used "placeholder" for things that are part of the URL path. But... I guess there's no need.

Reply:
Yeah, I think `placeholder` might be confusing and the `Query Parameters` and `Request Body` sections means we can easily use `Parameter` as a term here.